java-practices.iml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Why have you committed the IDE specific files to the main repo?
put them in .gitigonre file.